frame/tips/src/tests.rs

```diff
@@ -195,57 +283,101 @@ fn tip_hash() -> H256 {
+			Error::<Test, Instance1>::AlreadyKnown
 		);
 
 		let h = tip_hash();
-		assert_ok!(Tips::tip(Origin::signed(10), h.clone(), 10));
-		assert_ok!(Tips::tip(Origin::signed(11), h.clone(), 10));
-		assert_ok!(Tips::tip(Origin::signed(12), h.clone(), 10));
-		assert_noop!(Tips::tip(Origin::signed(9), h.clone(), 10), BadOrigin);
+		assert_ok!(Tips1::tip(Origin::signed(10), h.clone(), 10));
+		assert_ok!(Tips1::tip(Origin::signed(11), h.clone(), 10));
+		assert_ok!(Tips1::tip(Origin::signed(12), h.clone(), 10));
+		assert_noop!(Tips1::tip(Origin::signed(9), h.clone(), 10), BadOrigin);
 		System::set_block_number(2);
-		assert_ok!(Tips::close_tip(Origin::signed(100), h.into()));
+		assert_ok!(Tips1::close_tip(Origin::signed(100), h.into()));
 		assert_eq!(Balances::reserved_balance(0), 0);
 		assert_eq!(Balances::free_balance(0), 102);
 		assert_eq!(Balances::free_balance(3), 8);
+
+		Balances::make_free_balance_be(&Treasury2::account_id(), 111);
+		assert_ok!(Tips2::report_awesome(Origin::signed(0), b"awesome.dot".to_vec(), 3));
+		assert_eq!(Balances::reserved_balance(0), 12);
+		assert_eq!(Balances::free_balance(0), 90);
+
+		// other reports don't count.
+		assert_noop!(
+			Tips2::report_awesome(Origin::signed(1), b"awesome.dot".to_vec(), 3),
+			Error::<Test, Instance2>::AlreadyKnown
+		);
+
+		let h = tip_hash();
+		assert_ok!(Tips2::tip(Origin::signed(10), h.clone(), 10));
+		assert_ok!(Tips2::tip(Origin::signed(11), h.clone(), 10));
+		assert_ok!(Tips2::tip(Origin::signed(12), h.clone(), 10));
+		assert_noop!(Tips2::tip(Origin::signed(9), h.clone(), 10), BadOrigin);
+		System::set_block_number(3);
+		assert_ok!(Tips2::close_tip(Origin::signed(100), h.into()));
+		assert_eq!(Balances::reserved_balance(0), 0);
+		assert_eq!(Balances::free_balance(0), 104);
+		assert_eq!(Balances::free_balance(3), 16);
 	});
 }
 
 #[test]
 fn report_awesome_from_beneficiary_and_tip_works() {
 	new_test_ext().execute_with(|| {
-		Balances::make_free_balance_be(&Treasury::account_id(), 101);
-		assert_ok!(Tips::report_awesome(Origin::signed(0), b"awesome.dot".to_vec(), 0));
+		Balances::make_free_balance_be(&Treasury1::account_id(), 101);
+		assert_ok!(Tips1::report_awesome(Origin::signed(0), b"awesome.dot".to_vec(), 0));
 		assert_eq!(Balances::reserved_balance(0), 12);
 		assert_eq!(Balances::free_balance(0), 88);
 		let h = BlakeTwo256::hash_of(&(BlakeTwo256::hash(b"awesome.dot"), 0u128));
-		assert_ok!(Tips::tip(Origin::signed(10), h.clone(), 10));
-		assert_ok!(Tips::tip(Origin::signed(11), h.clone(), 10));
-		assert_ok!(Tips::tip(Origin::signed(12), h.clone(), 10));
+		assert_ok!(Tips1::tip(Origin::signed(10), h.clone(), 10));
+		assert_ok!(Tips1::tip(Origin::signed(11), h.clone(), 10));
+		assert_ok!(Tips1::tip(Origin::signed(12), h.clone(), 10));
 		System::set_block_number(2);
-		assert_ok!(Tips::close_tip(Origin::signed(100), h.into()));
+		assert_ok!(Tips1::close_tip(Origin::signed(100), h.into()));
```

Comment:
Please do not modify all the tests like this. Way too much here.

You just need 1 more new test which simply shows that the pallets are connected to one another. Some really basic stuff.